app/services/update_account_service.rb

```diff
@@ -28,7 +28,13 @@ def authorize_all_follow_requests(account)
   end
 
   def check_links(account)
-    VerifyAccountLinksWorker.perform_async(account.id) if account.fields.any?(&:requires_verification?)
+    return unless account.fields.any?(&:requires_verification?)
+
+    if account.local?
+      VerifyAccountLinksWorker.perform_async(account.id)
+    else
+      VerifyAccountLinksWorker.perform_in(rand(0..5).minutes.to_i, account.id)
```

Comment:
This will still concentrate large numbers of requests on specific minutes as the randomization is only between the numbers 0 and 5. You might want to randomize on seconds.

Reply:
Ah, good catch!